startDevelopmentStack.sh

```diff
@@ -8,9 +8,12 @@ docker login ghcr.io -u $GITHUB_USER -p $GITHUB_TOKEN
 mkdir -p ./local/certs
 scp ubuntu@dataland-letsencrypt.duckdns.org:/etc/letsencrypt/live/dataland-local.duckdns.org/* ./local/certs
 
+rm ./*github_env.log
+find ./build-utils/ -name "rebuild*.sh" -exec bash -c 'eval "$1"' shell {} \;
 
-./gradlew dataland-keycloak:dataland_theme:login:buildTheme --no-daemon --stacktrace
-./gradlew dataland-frontend:generateAPIClientFrontend --no-daemon --stacktrace
+set -o allexport
```

Comment:
das könnte man sich sparen wenn man in base_rebuild_single_docker_image.sh: `echo "export ${docker_image_name^^}_VERSION=$input_sha1" >> ./${BUILD_SCRIPT:-default}_github_env.log` schreibt

Reply:
siehe https://github.com/d-fine/Dataland/pull/126#discussion_r1017829836